Changelog.md

```diff
@@ -11,6 +11,7 @@ Compiler Features:
  * Commandline Interface: option ``--pretty-json`` works also with ``--standard--json``.
  * SMTChecker: Unproved targets are hidden by default, and the SMTChecker only states how many unproved targets there are. They can be listed using the command line option ``--model-checker-show-unproved`` or the JSON option ``settings.modelChecker.showUnproved``.
  * SMTChecker: new setting to enable/disable encoding of division and modulo with slack variables. The command line option is ``--model-checker-div-mod-slacks`` and the JSON option is ``settings.modelChecker.divModWithSlacks``.
+ * Type System: Introduced global ``block.basefee`` for retrieving the base fee of the current block.
```

Comment:
Maybe also add the added Yul function?

Reply:
Added this below.

Reply:
Isn't this a language feature instead of a compiler feature? Both of them?